### DISCUSSION TITLE
[SSR] Error transforming ./node_modules/@pinia/nuxt/templates/plugin.js

### DISCUSSION THREAD

**Discussion by matthewhardern** [vuejs/pinia#734] [Help and Questions] [answered]
### Steps to reproduce the behavior

`npx nuxi init test-nuxt3`

`npm i pinia`

`npm i @pinia/nuxt`

add nuxt config

```
export default defineNuxtConfig({
    modules: [
        '@pinia/nuxt',
    ]
})
```

I get ` WARN  [SSR] Error transforming ./node_modules/@pinia/nuxt/templates/plugin.js: Error: Missing "./templates/plugin.js" export in "@pinia/nuxt" package`

### Expected behavior

Not to error

### Actual behavior

SSR error

**Comment by posva** [comment]
It has to be put in BuildModules

**Comment by matthewhardern** [comment]
I still get the same error if I do

**Comment by posva** [comment, marked as answer]
Here you have an example with Nuxt 3: https://github.com/piniajs/example-nuxt-3

**Comment by matthewhardern** [comment]
this is the latest version `3.0.0-27245590.7c3b488` it breaks on this version

**Reply by posva** [reply]
alright, let me check. At least `3.0.0-27245396.550a9f2` is working!
I think I see where the issue is coming from

**Reply by posva** [reply]
A patch should be up!

**Reply by matthewhardern** [reply]
@posva much appreciated, the library is awsome btw
